3.10/indexing-working-with-indexes.md

```diff
@@ -161,6 +161,19 @@ The **estimates** attribute is optional and defaults to *true* if not set. It wi
 have no effect on indexes other than *persistent* (with *hash* and *skiplist*
 being mere aliases for *persistent* nowadays).
 
+**cacheEnabled** can be *true* or *false* and is supported by indexes of type
+*persistent*. The attribute controls whether an extra in-memory hash cache is
+created for the index. The hash cache can be used to speed up index lookups.
+The cache can only be used for queries that look up all index attributes via
+an equality lookup (`==`). The hash cache cannot be used for range scans, 
+partial lookups or sorting.
+The cache will populated lazily upon reading data from the index. Writing data
+into the collection or updating existing data will invalidate entries in the
+cache. The cache may have a negative effect on performance in case index values
+are updated more often than they are read. 
+**cacheEnabled** defaults to *false* and should only be used for indexes that
+are known to benefit from an extra layer of caching.
```

Comment:
Should potentially reference this feature in https://www.arangodb.com/docs/stable/aql/execution-and-performance.html. However, we don't have a proper "how to optimize AQL queries" guide in the docs yet where it would fit in well.